packages/core/src/main.js

```diff
@@ -0,0 +1,7 @@
+import '@department-of-veterans-affairs/web-components/dist/component-library/component-library.css';
+import webComponentDocs from '@department-of-veterans-affairs/web-components/component-docs';
+
+export * from '@department-of-veterans-affairs/react-components';
+export * from '@department-of-veterans-affairs/web-components';
+export * from '@department-of-veterans-affairs/web-components/loader';
+export {webComponentDocs};
```

Comment:
This is what the `core` bundle will contain:

1. The css for the web components that defines all of the top-level CSS variables
1. The JSON for web component docs that gets used in storybook
1. The files exported by `index.js` in `react-components`
1. The exports from `web-components/dist`
1. The exports from `web-components/loader` so that we get `applyPolyfills`.

Reply:
Are we importing `webComponentDocs` and then exporting it separately because we want to re-export it as a named export?

I guess my question is if we could do this:
```js
export { default as webComponentDocs } from '@department-of-veterans-affairs/web-components/component-docs';
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
I was getting some syntax errors with how I was trying to import and export it at once, but I'll try that line and see what happens.